### PR TITLE
docs: DOC-1385

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR adds release notes for an upcoming patch.

## Changed Pages

💻 [Preview URL for Page]()

## Jira Tickets

🎫 [DOC-1385](https://spectrocloud.atlassian.net/browse/DOC-1385)

## Backports

Can this PR be backported?

- [x] Yes. _Remember to add the relevant backport labels to your PR._
- [ ] No. _Please leave a short comment below about why this PR cannot be backported._

[DOC-1385]: https://spectrocloud.atlassian.net/browse/DOC-1385?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/release-notes/release-notes.md [hunk 1]
```diff
@@ -11,6 +11,48 @@ tags: ["release-notes"]
...
 
+## Sept 25, 2024 - Release 4.4.20
+
+### Improvements
+
+- Palette's message communication channel between clusters and the management plane has been updated to support gRPC
+  over websockets. Palette agents will automatically fall back to web sockets if the gRPC connection cannot be
+  established using the default HTTP/2 protocol. This change improves the reliability of the communication channel
+  between the agent and the management plane. Environments with network proxies that reject HTTP/2 connections can now
+  connect as the connection is transcoded to the HTTP/1.1 protocol.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/release-notes/release-notes.md:22]:
```suggestion
- Palette's message communication channel between clusters and the management plane has been updated to support gRPC
  over WebSocket. Palette agents will automatically fall back to a WebSocket connection if the gRPC connection cannot be
  established with the management plane using the default HTTP/2 protocol. This change improves the reliability of the communication channel
  between the agent and the management plane. Environments with network proxies that reject HTTP/2 connections can now
  connect as the connection is transcoded to the HTTP/1.1 protocol.
```

File: docs/docs-content/release-notes/release-notes.md [hunk 2]
```diff
@@ -11,6 +11,48 @@ tags: ["release-notes"]
...
+  over WebSocket. Palette agents will automatically fall back to a WebSocket connection if the gRPC connection cannot be
+  established with the management plane using the default HTTP/2 protocol. This change improves the reliability of the
+  communication channel between the agent and the management plane. Environments with network proxies that reject HTTP/2
+  connections can now connect as the connection is transcoded to the HTTP/1.1 protocol.
+
+- Local UI now supports selecting network interfaces by name.
+
+- Local UI has improved validation when assigning static IP addresses to network interfaces.
+
+- The heartbeat mechanism for Palette deployed clusters has been improved to improve performance and reliability.
```
Comment (lennessyy) [inline docs/docs-content/release-notes/release-notes.md:28]:
```suggestion
- The heartbeat mechanism for Palette deployed clusters has been improved for better performance and reliability.
```

Comment (lennessyy) [reply docs/docs-content/release-notes/release-notes.md:28]: maybe? "improved to improve" sounds a little awkward

File: docs/docs-content/release-notes/release-notes.md [hunk 3]
```diff
@@ -11,6 +11,48 @@ tags: ["release-notes"]
...
+
+#### Bug Fixes
+
+- Fixed an issue where imported cluster profiles defaulted to incorrect registry type.
+
+- Fixed an issue where Local UI TUI was not displaying network interfaces that had no IP addresses assigned.
```
Comment (lennessyy) [inline docs/docs-content/release-notes/release-notes.md:34]:
```suggestion
- Fixed an issue where Palette TUI was not displaying network interfaces that had no IP addresses assigned.
```
If it's the TUI during Edge host setup, i think it's called Palette TUI

File: docs/docs-content/release-notes/release-notes.md [hunk 4]
```diff
@@ -11,6 +11,48 @@ tags: ["release-notes"]
...
+
+- Resolved an issue where the Palette UI was erroring out when reviewing pack layers during a PCG upgrade.
+
+- Resolved an issue where the Local UI username and password field validation was not working as expected.
+
+- Fixed an issue with updating the node port for the Edge Harbor pack. The node port is now correctly updated when
+  changing the value in the Harbor pack.
+
+- Resolved an issue where kubeadm based Kubernetes distributions were unable to initialize on clusters with multiple
+  Network Interfaces (NIC).
```
Comment (lennessyy) [inline docs/docs-content/release-notes/release-notes.md:54]:
```suggestion
  Network Interface Cards (NIC).
```

**Review by lennessyy — APPROVED** [review]

**Pull request merged**